src/domain/select_local_cameras.py

```diff
@@ -0,0 +1,126 @@
+            label = QLabel(f'{camera_info.vid}')
+            label.setObjectName("label_cameraVid"+camera_index)
+            self.ui.gridLayout_localCameras.addWidget(label, row, 4)
+            # Camera ID
+            id_line_edit = QLineEdit()
+            lnedit_obj_name = "lineEdit_cameraID"+camera_index
+            id_line_edit.setObjectName(lnedit_obj_name)
+            id_line_edit.textChanged.connect(self.validate_cameras_selection)
+            self.ui.gridLayout_localCameras.addWidget(id_line_edit, row, 5)
+            # Test button to preview video (and detection if enabled)
+            test_button = QPushButton('Test video')
+            test_button.setObjectName("button_camera"+camera_index)
+            self.ui.gridLayout_localCameras.addWidget(test_button, row, 6)
+            #TODO: connect slot test_button.clicked.connect()
+
+            row = row +1
+        self.ui.gridLayout_localCameras.setAlignment(Qt.AlignmentFlag.AlignTop)
+
+    def initialize_usb_cameras(self):
+        """Method to initialize attributes from existing camera configuration (if any)."""
+        if not self.cameras_list:
+            return
+ 
+        # Initialize camera checkboxes (if any was selected before) and IDs
+        camera_number = len(enumerate_cameras(cv2.CAP_MSMF))
+        for i in range(camera_number):
+            checkbox_obj_name = "checkBox_camera"+i
+            id_line_edit_name = "label_cameraName"+i
+            checkbox = self.ui.gridLayout_localCameras.findChild(QCheckBox, checkbox_obj_name)
+            line_edit = self.ui.gridLayout_localCameras.findChild(QLineEdit, id_line_edit_name)
+            for camera in self.cameras_list:
+                if camera.id == line_edit.text():
+                    checkbox.setChecked(camera.is_enabled)
+                    camera.id = i
+
+    def initialize_detection_params(self):
+        """Method to initialize detection parameters in UI."""
+
+        self.ui.lineEdit_treshold.setText(str(Camera.detection_params['treshold']))
+        self.ui.lineEdit_minContourArea.setText(str(Camera.detection_params['min_contour_area']))
+        self.ui.lineEdit_detectionTreshold.setText(str(
+            Camera.detection_params['detection_per_second']))
+
+    def validate_cameras_selection(self):
+        """This method validated the selected cameras."""
+
+        valid = True
+        for camera_info in enumerate_cameras(cv2.CAP_MSMF):
+            camera_index = str(camera_info.index)
+            checkbox_obj_name = "checkBox_camera"+camera_index
+            id_line_edit_name = "label_cameraName"+camera_index
+            checkbox = self.ui.formLayout_caleraList.findChild(QCheckBox, checkbox_obj_name)
+            line_edit = self.ui.formLayout_caleraList.findChild(QLineEdit, id_line_edit_name)
+            if checkbox and checkbox.isChecked() and not line_edit.text():
+                self.ui.label_errorMessage.setText("Please add ID for Camera %s", camera_index)
```

Comment:
There's no string interpolation for this: "Please add ID for Camera %s", camera_index
Is it something which the method does internally, like the logging methods?